### ISSUE TITLE
Typecheck error in `index.d.ts`

### ISSUE THREAD

**Issue by kirill-galimov-plutotv** [ValentinH/jest-fail-on-console#22]
There is a typecheck error in `index.d.ts` file. 

"jest-fail-on-console": `2.4.1`
"typescript": `4.5.5`

```typescript
  errorMessage?: (
    methodName: 'assert' | 'debug' | 'error' | 'info' | 'log' | 'warn',
    bold: (string) => string // Parameter has a name but no type. Did you mean 'arg0: string'?
  ) => string
```

![image](https://user-images.githubusercontent.com/101738595/169849717-1000c461-addf-4ab8-b93b-b57ca6231dec.png)

**Comment by ValentinH** [comment]
Good catch, would you like to submit a PR to fix it?

**Comment by KirillGalimov** [comment]
@ValentinH I can submit a PR to fix it, how can I get permissions?

**Comment by ValentinH** [comment]
You just have to fork this repository, make the changes there and submit a PR.
I think you could even do it by editing the right file from Github UI

**Comment by KirillGalimov** [comment]
@ValentinH thanks! done.

**Comment by ValentinH** [comment]
Fixed by #23
